source/elements/oneTBB/source/task_scheduler.rst

```diff
@@ -46,6 +46,7 @@ Task Group
 
    task_scheduler/task_group/task_group_cls.rst
    task_scheduler/task_group/task_group_status_enum.rst
+   task_scheduler/task_group/task_handle.rst
```

Comment:
Please, update the copyright year for this file.

Reply:
updated